packages/stack/contracts-manager/src/contract.ts

```diff
@@ -18,6 +18,8 @@ export default class Contract {
   public silent?: boolean = false;
   public track?: boolean = true;
   public deploy?: boolean = true;
+  public privateFor?: string[] = undefined;
+  public privateFrom?: string = undefined;
```

Comment:
It should be enough to do `privateFor?: string[]`; etc. without assigning `undefined`. Every variable that is not intialized is `undefined`.

However, if the type annotation is there, and we **do** want to initialize, we should do it with a value that matches the type. So..

```
privateFor?: string[] = [];
proviateFrom? = ''; <-- this probably doesn't make too much sense, so rather not initalize at all
```

Reply:
Good catch, thank you! Updated without initialisers.